src/VersionTool/Program.cs

```diff
@@ -225,13 +253,324 @@ private static int OnList(CommandOption pathOption)
                 {
                     SaveJObject(project.ProjectFilePath, root);
 
-                    Console.WriteLine($"Updated: {project.ProjectFilePath}");
+                    //Console.WriteLine($"Updated: {project.ProjectFilePath}");
```

Comment:
Oops, I was debugging something and add/removed some of the logging. Should have checked after I pushed my last commit.